### PR TITLE
Fix to cross thread operation bug

### LINKED ISSUE #76: Exception "Cross-thread operation not valid: Control 'progressBar' accessed from a thread other than the thread it was created on."
# Steps To Reproduce

1. Build latest commit from master (95aa1bb)
2. Run the Gui under debugger to make it easier to run into the exception
3. Load an assembly containing tests => works fine, i.e. discovers all tests as expected
4. Select a single test and select "Run" from context menu => Exception

# Exception Details
## Exception Message
"Cross-thread operation not valid: Control 'progressBar' accessed from a thread other than the thread it was created on."
## Stack Trace

   at System.Windows.Forms.Control.get_Handle()
   at System.Windows.Forms.Control.Invalidate(Boolean invalidateChildren)
   at System.Windows.Forms.Control.Refresh()
   at TestCentric.Gui.Controls.ColorProgressBar.OnForeColorChanged(EventArgs e) in C:\projects\testcentric\testcentric-gui\src\TestCentric\testcentric.gui\Controls\ProgressBar.cs:line 248
   at System.Windows.Forms.Control.set_ForeColor(Color value)
   at TestCentric.Gui.Controls.TestProgressBar.Initialize(Int32 testCount) in C:\projects\testcentric\testcentric-gui\src\TestCentric\testcentric.gui\Controls\ProgressBar.cs:line 366
   at TestCentric.Gui.Controls.TestProgressBar.<>c__DisplayClass7_0.<InitializeView>b__3(RunStartingEventArgs e) in C:\projects\testcentric\testcentric-gui\src\TestCentric\testcentric.gui\Controls\ProgressBar.cs:line 393

### PR DESCRIPTION
**Pull request by drSte**
fixes #76

### CODE REVIEW

File: src/TestCentric/testcentric.gui/Presenters/TestCentricPresenter.cs [hunk 1]
```diff
@@ -187,8 +187,8 @@ private void WireUpEvents()
                 //    //log.Warning("Unable to save result to {0}\n{1}", resultPath, ex.ToString());
                 //}
                          
-                if (e.Result.Outcome.Status == TestStatus.Failed)
-                    _view.Activate();
+                //if (e.Result.Outcome.Status == TestStatus.Failed)
+                //    _view.Activate();
```
Comment (CharliePoole) [inline src/TestCentric/testcentric.gui/Presenters/TestCentricPresenter.cs]: You're removing a feature here.

Comment (drSte) [reply src/TestCentric/testcentric.gui/Presenters/TestCentricPresenter.cs]: @CharliePoole ,
I will try to fix also that part

Comment (CharliePoole) [reply src/TestCentric/testcentric.gui/Presenters/TestCentricPresenter.cs]: Fix should be essentially the same.

**Comment by drSte** [comment]
@CharliePoole is it possible that you removed the code that is tested with the failing unit test with the squash of my last PR?

**Comment by CharliePoole** [comment]
Code changes look good and are definitely needed. It looks like the ProgressBarView was already fixed in response to a different issue.

**Pull request merged**